### PR TITLE
Improve Linking Nodes documentation example + add extended helpers

### PR DESCRIPTION
**Pull request by Vadorequest**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: 
- https://github.com/reaviz/reaflow/issues/47
- https://github.com/reaviz/reaflow/issues/42

## What is the new behavior?

Documentation improved.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Fixes https://github.com/reaviz/reaflow/issues/42

Removed unused storybook config.

### CODE REVIEW

**Comment by Vadorequest** [comment]
@amcdnl Ready for review :)

File: docs/Advanced/ExtendedUtils.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,177 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Docs/Advanced/Extended utils" />
```
Comment (amcdnl) [inline docs/Advanced/ExtendedUtils.stories.mdx:3]:
```suggestion
<Meta title="Docs/Advanced/Custom Utils" />
```

File: docs/Advanced/Utils.stories.mdx [hunk 2]
```diff
@@ -12,6 +12,12 @@ helpers are broken up into two categories:
 - CRUD - Dealing with manipulating nodes/edges
 - Graph Dealing with traversing the graph
 
+> Those helpers are very generic and meant to help people getting started, but they're not meant to cover specific use-cases.
+>
+> You should copy them and adapt them to your own needs if they don't cover your needs. [See discussion](https://github.com/reaviz/reaflow/issues/47#issuecomment-775919579).
+>
+> There are advanced examples in [Extended utils](/?path=/story/docs-advanced-extended-utils--page) page.
```
Comment (amcdnl) [inline docs/Advanced/Utils.stories.mdx:19]:
```suggestion
> There are advanced examples in [Custom Utils](/?path=/story/docs-advanced-custom-utils--page) page.
```

**Review by amcdnl — CHANGES_REQUESTED** [review]

**Comment by amcdnl** [comment]
Just a name tweak other LGTM

**Comment by Vadorequest** [comment]
Applied, I didn't know I could apply suggestion directly from github, it's handy!

I gues you can merge now :)

**Pull request merged**